tests/integration/properties/test_material.py

```diff
@@ -0,0 +1,95 @@
+import os
+from copy import deepcopy
+from typing import Dict, List
+
+from express.parsers.apps.espresso.parser import EspressoParser
+from express.parsers.apps.vasp.parser import VaspParser
+from express.properties.material import Material
+from tests.fixtures.data import SI as data
+from tests.integration import IntegrationTestBase
+
+
+class MaterialTest(IntegrationTestBase):
+    def setUp(self):
+        super(MaterialTest, self).setUp()
+
+    def tearDown(self):
+        super(MaterialTest, self).setUp()
```

Comment:
this should be `super(MaterialTest, self).tearDown()` right?

Reply:
I think that is also incorrect in other places (e.g. `integration/__init__.py`).